api/Rakefile

```diff
@@ -52,3 +53,13 @@ namespace :db do
     sh "rm #{databases['production']['database']}"
   end
 end
+
+namespace :log do
+  log_config = YAML.load_file('config/log.yml')
+  task :create do
+    puts 'Creating API logs …'
+    sh 'mkdir /var/www/log' unless File.directory?('/var/www/log')
```

Comment:
Have a look at this:

Folder creation (recursively)

```ruby
require 'fileutils'

dirname = File.dirname(some_path)
unless File.directory?(dirname)
  FileUtils.mkdir_p(dirname)
end
```

File creation for writing

```ruby
out_file = File.new("out.txt", "w")
```